resources/ec2_security_groups.go

```diff
@@ -475,3 +481,9 @@ func fetchEc2SecurityGroupIpPermissionsEgressUserIdGroupPairs(ctx context.Contex
 	res <- securityGroupIpPermissionEgress.UserIdGroupPairs
 	return nil
 }
+
+func resolveSGArn(_ context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

Comment:
This code comes from here:
https://github.com/cloudquery/cq-provider-aws/blob/c47fb629c96a50e960a036d516ce83352b0df79a/resources/ec2_ebs_volumes.go#L177-L181

Reply:
This should also do an assertion `ebs, ok` check.